pkg/apis/config/v1beta1/zz_generated.defaults.go

```diff
@@ -28,22 +28,5 @@ import (
 // Public to allow building arbitrary schemes.
 // All generated defaulters are covering - they call all nested defaulters.
 func RegisterDefaults(scheme *runtime.Scheme) error {
-	scheme.AddTypeDefaultingFunc(&CapacitySchedulingArgs{}, func(obj interface{}) { SetObjectDefaults_CapacitySchedulingArgs(obj.(*CapacitySchedulingArgs)) })
```

Comment:
Why remove this? For a follow-up exercising v1alpha1 args?

Reply:
No idea. It's generated code. Restored it to the original version.

Reply:
No idea, it's code generated by code-generated. restored it.